modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -73,6 +73,17 @@ REMOVE n[$prop]
 
 | Added the ability to dynamically reference properties in xref:clauses/set.adoc#set-dynamically-a-property[SET] and xref:clauses/remove.adoc#remove-remove-a-property-dynamically[REMOVE] clauses.
 
+a|
```

Comment:
might need to switch which version it's added in depending on when it gets merged into product